tests/components/whirlpool/test_sensor.py

```diff
@@ -147,9 +147,19 @@ async def test_washer_sensor_values(
     assert state.state == thetimestamp.isoformat()
 
     state_id = f"{entity_id.split('_')[0]}_detergent_level"
+    registry = entity_registry.async_get(hass)
+    entry = registry.async_get(state_id)
+    assert entry
+    assert entry.disabled
+    assert entry.disabled_by is entity_registry.RegistryEntryDisabler.INTEGRATION
+    update_entry = registry.async_update_entity(
+        entry.entity_id, **{"disabled_by": None}
+    )
+    await hass.async_block_till_done()
```

Comment:
Reload the config entry so we can test the state with the entity enabled below.

Reply:
I've added the reload, but the test is failing and I don't understand why.  I can reload fine (without errors that I see) in runtime and enable the default disabled entity, but the test is throwing an iteration error on the sensor during the reload that I don't understand.

Reply:
Figured it out...